packages/table/src/Table.story.tsx

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { boolean } from '@storybook/addon-knobs';
+import Table from '.';
+import { Row, Cell, TableHeader, HeaderRow, DataType } from '.';
+import { defaultData, multiRowData } from './storybookdata';
+
+interface DemoDataInterface {
+  name: string;
+  age: number;
+  color: string;
+  location: string;
+}
+
+storiesOf('Table', module)
+  .add('Default', () => (
+    <Table
+      selectable={boolean('isSelectable', true)}
+      data={defaultData}
+      columns={[
```

Comment:
Do you think its possible to also accept a Fragment here so we could do:

```jsx
columns={
   <>
       <TableHeader />
        Location
   </>
}
```

Is that possible?

Reply:
Yes possible, done!